### ISSUE TITLE
[BUG] using mariaDB causes Aborted connection xx to db: 'db' user: 'bunkerweb' host: '172.20.0.4' (Got an error reading communication packets)

### ISSUE THREAD

**Issue by chrismade** [bunkerity/bunkerweb#557]
**Description**
when using mariaDB (e.g. in autoconf) I see a lot of messages
`Aborted connection xx to db: 'db' user: 'bunkerweb' host: '172.20.0.4' (Got an error reading communication packets)`
in the db container and further
`CONFIG - ⚠️  - Can't connect to database, retrying in 5 seconds ...`
in the other containers that rely on / open connections to the DB

**How to reproduce**
Follow the instructions in 
https://docs.bunkerweb.io/1.5.0/integrations/#docker-autoconf
1.) copy/paste the provided "docker-compose.yml" 
2.) fix the blank SERVER_NAME= by providing a server name e.g. SERVER_NAME=sorry.5.75.248.196.nip.io instead of leaving it blank (which will cause errors)
3.) docker compose up
4.) see logfiles

**Logs**

```
chris-bw-db-1         | 2023-07-11 19:19:17 11 [Warning] Aborted connection 11 to db: 'db' user: 'bunkerweb' host: '172.20.0.4' (Got an error reading communication packets)
chris-bw-scheduler-1  | [2023-07-11 19:19:17] - SCHEDULER - ℹ️  - Successfully updated database for the job mmdb-country from plugin jobs
chris-bw-scheduler-1  | [2023-07-11 19:19:17] - SCHEDULER - ℹ️  - Executing job mmdb-asn from plugin jobs ...
chris-bw-scheduler-1  | [2023-07-11 19:19:17] - BUNKERNET - ℹ️  - Database connection established
chris-bw-scheduler-1  | [2023-07-11 19:19:17] - BUNKERNET - ℹ️  - Successfully retrieved BunkerNet ID from db cache
chris-bw-scheduler-1  | [2023-07-11 19:19:17] - BUNKERNET - ℹ️  - BunkerNet list is already in cache, skipping download...
chris-bw-db-1         | 2023-07-11 19:19:17 12 [Warning] Aborted connection 12 to db: 'db' user: 'bunkerweb' host: '172.20.0.4' (Got an error reading communication packets)
chris-bw-scheduler-1  | [2023-07-11 19:19:17] - SCHEDULER - ℹ️  - Successfully updated database for the job bunkernet-data from plugin bunkernet
chris-bw-scheduler-1  | [2023-07-11 19:19:17] - SCHEDULER - ℹ️  - Executing job greylist-download from plugin greylist ...
chris-bw-autoconf-1   | [2023-07-11 19:19:17] - CONFIG - ⚠️  - Can't connect to database, retrying in 5 seconds ...
chris-bw-scheduler-1  | [2023-07-11 19:19:18] - SELF-SIGNED - ℹ️  - Database connection established
chris-bw-scheduler-1  | [2023-07-11 19:19:18] - SCHEDULER - ℹ️  - Successfully updated database for the job self-signed from plugin selfsigned
chris-bw-scheduler-1  | [2023-07-11 19:19:18] - LETS-ENCRYPT.NEW - ℹ️  - Database connection established
chris-bw-scheduler-1  | [2023-07-11 19:19:18] - LETS-ENCRYPT.NEW - ℹ️  - Database connection established
chris-bw-scheduler-1  | [2023-07-11 19:19:18] - LETS-ENCRYPT.NEW - ℹ️  - Successfully retrieved Let's Encrypt data from db cache
chris-bw-scheduler-1  | [2023-07-11 19:19:18] - LETS-ENCRYPT.NEW - ℹ️  - Successfully saved Let's Encrypt data to db cache
chris-bw-scheduler-1  | [2023-07-11 19:19:19] - SCHEDULER - ℹ️  - Successfully updated database for the job certbot-new from plugin letsencrypt
chris-bw-scheduler-1  | [2023-07-11 19:19:19] - SCHEDULER - ℹ️  - Executing job certbot-renew from plugin letsencrypt ...
chris-bw-scheduler-1  | [2023-07-11 19:19:19] - JOBS.MMDB-ASN - ℹ️  - Database connection established
chris-bw-scheduler-1  | [2023-07-11 19:19:19] - JOBS.MMDB-ASN - ℹ️  - asn.mmdb is already in cache, skipping download...
chris-bw-db-1         | 2023-07-11 19:19:19 16 [Warning] Aborted connection 16 to db: 'db' user: 'bunkerweb' host: '172.20.0.4' (Got an error reading communication packets)
```

it looks like the database connection could be established - just to be aborted soon thereafter

I already tried a few well-known patterns to fix this warning (which is actually an error if the connection is dropped and the consuming application is unable to re-connect) at start of the mariaDB container -- but none of those fixed the root cause

```
    - --transaction-isolation=READ-COMMITTED
    - --log-bin
    - --log-bin=mysqld-bin
    - --character-set-server=utf8mb4
    - --collation-server=utf8mb4_unicode_ci
    - --max-connections=1000
    - --skip-name-resolve
    - --wait-timeout=28800 # // 28800 sec = 8 hours
    - --max_allowed_packet=1073741824
    - --binlog-format=ROW
    - --innodb_use_native_aio=OFF
```

**Comment by TheophileDiot** [comment]
@chrismade could you provide us your docker-compose as well please ? Thank you

**Comment by chrismade** [comment]
as requested ... here is my docker-compose.yml (sorry I was unaware that SERVER_NAME= needs to be empty - but it looks like the db connection abort may have a different root cause):
 
```
version: "3.5"

services:
  bunkerweb:
    image: bunkerity/bunkerweb:1.5.0
    ports:
      - 8080:8080
      - 8443:8443
    labels:
      - "bunkerweb.INSTANCE"
    environment:
      - MULTISITE=yes
      - API_WHITELIST_IP=127.0.0.0/8 10.20.30.0/24
      - SERVER_NAME=sorry.5.77.222.99.nip.io www.5.77.222.99.nip.io clock.5.77.222.99.nip.io
      - www.5.77.222.99.nip.io_SERVER_NAME=www.5.77.222.99.nip.io
      - www.5.77.222.99.nip.io_AUTO_LETS_ENCRYPT=no
      - www.5.77.222.99.nip.io_EMAIL_LETS_ENCRYPT=privacy@something.com
      - www.5.77.222.99.nip.io_USE_LETS_ENCRYPT_STAGING=no
      - sorry.5.77.222.99.nip.io_SERVER_NAME=sorry.5.77.222.99.nip.io
      - sorry.5.77.222.99.nip.io_AUTO_LETS_ENCRYPT=no
      - sorry.5.77.222.99.nip.io_EMAIL_LETS_ENCRYPT=privacy@something.com
      - sorry.5.77.222.99.nip.io_USE_LETS_ENCRYPT_STAGING=no
      - clock.5.77.222.99.nip.io_SERVER_NAME=clock.5.77.222.99.nip.io
      - clock.5.77.222.99.nip.io_AUTO_LETS_ENCRYPT=no
      - clock.5.77.222.99.nip.io_EMAIL_LETS_ENCRYPT=privacy@something.com
      - clock.5.77.222.99.nip.io_USE_LETS_ENCRYPT_STAGING=no
    volumes:
      - /home/chris/bwdata:/data
      - /home/chris/www:/var/www/html
    networks:
      - bw-universe
      - bw-services

  bw-scheduler:
    image: bunkerity/bunkerweb-scheduler:1.5.0
    depends_on:
      - bunkerweb
      - bw-docker
    volumes:
      - /home/chris/bwdata:/data
    environment:
      - DOCKER_HOST=tcp://bw-docker:2377
    networks:
      - bw-universe
      - bw-docker

  bw-docker:
    image: tecnativa/docker-socket-proxy
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock:ro
    environment:
      - CONTAINERS=1
    networks:
      - bw-docker

networks:
  bw-universe:
    name: bw-universe
    ipam:
      driver: default
      config:
        - subnet: 10.20.30.0/24
  bw-services:
    name: bw-services
  bw-docker:
    name: bw-docker
```

**Comment by fl0ppy-d1sk** [comment]
Hello @chrismade,

You are talking about autoconf but your example is classical docker.

Anyway, we found a way to remove the `Got an error reading communication packets` warnings. Fix will be integrated into the 1.5.1 release.

**Comment by fl0ppy-d1sk** [comment]
Hello @chrismade,

It should be fixed in the last 1.5.1 release. Thanks for your report and don't hesitate to open a new issue if needed.
